<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>206</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QDoubleSpinBox" name="PiTimeSpinBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>38</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>10</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>2Pi Time</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pi Time</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Pi/2 Time</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>39</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>us</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
